tests/dummy/app/app.js

```diff
@@ -1,6 +1,6 @@
 import Ember from 'ember'
 const {Application} = Ember
-import Resolver from 'ember-resolver'
+import Resolver from './resolver'
```

Comment:
So the reasoning is, "this is how the cool kids do it"? Do we happen to know *why* it's done that way? Seems kinda pointless unless we're gonna change the behavior. Did you see this documented somewhere? Or was there some deprecation warning or something?